packages/gateway/src/server.ts

```diff
@@ -89,6 +89,17 @@ export async function createServer() {
   const routes = getRoutes()
   app.route(routes)
 
+  app.ext('onRequest', (req, h) => {
```

Comment:
I think there's two benefints in doing it the other way around, so redireding `/v1/*` to the underlying routes:

- When we change the version, we only need to change it here and do not have to make a change to all of our route definitions
- We could more easily make this version number dynamic so it's read for instance from package.json's version for instance

Con I can think of is of course that you somehow have to trick Swagger to list out all routes as /{VERSION}/{real path}

Reply:
swagger has this option which we can use: https://github.com/hapi-swagger/hapi-swagger/blob/master/usageguide.md#rewriting-paths-and-groupings so let's just go with this then